deployments/traefik.go

```diff
@@ -116,6 +116,7 @@ func (k Traefik) apply(ctx context.Context, c *kubernetes.Cluster, ui *termui.UI
 	if err != nil {
 		return err
 	}
+	loadBalancerIP := options.GetStringNG("ingress-service-ip")
```

Comment:
When traefik installation is skipped, either explicitly (--skip-traefik) or implicitly (because it's found), this option will be ignored. I think we should show a warning to the user to ensure they know their setting wasn't respected. E.g.

```
Skipping Traefik Ingress deployment by user request. The `ingress-service-ip` setting will be ignored.
```

Reply:
Oh, you already do that here: https://github.com/epinio/epinio/pull/653/files#diff-c0a23dc7003156100c2e86d23fcfe738c2037b8e32c5d95c956baa7f120a0ec3R156
I think we are still missing the implicit skip though.

Reply:
Done.